app/uk/gov/hmrc/hmrcemailrenderer/templates/mandate/mandateAgentActivates.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, params("service").toString) {
+
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Your agent has now accepted your request, they now has the authority to act for you on ATED.</p>
+
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">From HMRC</p>
+}
```

Comment:
We use to add some information about why they are receiving this email, it helps to avoid the recipient to mark it as spam, for example: https://github.com/hmrc/hmrc-email-renderer/blob/master/app/uk/gov/hmrc/hmrcemailrenderer/templates/digitalcontact/newMessageAlertSA316_A_detailed.scala.html#L8-L9